src/main/java/org/alliancegenome/curation_api/model/entities/BiologicalEntity.java

```diff
@@ -26,5 +37,8 @@ public class BiologicalEntity extends CurieAuditedObject {
     @JsonView({View.FieldsOnly.class})
     private NCBITaxonTerm taxon;
     
+    //@OneToMany(mappedBy = "diseaseGeneticModifier", cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
+    //private    List<DiseaseAnnotation> diseaseGeneticModifiers;
+    
```

Comment:
Are these coming back? There is the exclude in the @ToString

Reply:
No, I will remove those and the exclude

Reply:
Done